test/parallel/test-buffer-arraybuffer.js

```diff
@@ -43,12 +43,16 @@ assert.throws(function() {
   Buffer.from(new AB());
 }, TypeError);
 
-// write{Double,Float}{LE,BE} with noAssert should not crash, cf. #3766
+// https://github.com/nodejs/node/issues/3766
```

Comment:
Since this GitHub issue is specifically about behavior when `noAssert` is true, I think the URL/comment can be removed.